src/Analysis/Ast/Impl/Analyzer/ModuleWalker.cs

```diff
@@ -258,6 +260,10 @@ public ModuleWalker(IServiceContainer services, IPythonModule module)
 
                         var memberType = member?.GetPythonType();
                         var stubMemberType = stubMember.GetPythonType();
+
+                        if (builtins.Equals(memberType?.DeclaringModule) || builtins.Equals(stubMemberType.DeclaringModule)) {
```

Comment:
Why not the `stubMemberType.IsBuiltin`?

Reply:
Yep

Reply:
Actually no. We have a bit of a mess with what is `IsBuiltin` on the type. Ex `PythonModule` has `IsBuiltin => true` always. So does `PythonVariableModule`. Or `Union` when it has all types as builtin types... Modules assume they are builtin if `IsBuiltin => IsCompiled && ModulePath == null`.

This specific check is if member actually belongs to the `builtins` module. We should go clean this up, but it may be some other time.